lib/osf-components/addon/components/form-controls/radio-button/component.ts

```diff
@@ -0,0 +1,35 @@
+import { tagName } from '@ember-decorators/component';
+import { action } from '@ember-decorators/object';
+import Component from '@ember/component';
+import { assert } from '@ember/debug';
+
+import { ChangesetDef } from 'ember-changeset/types';
+import { layout } from 'ember-osf-web/decorators/component';
+import defaultTo from 'ember-osf-web/utils/default-to';
+
+import styles from './styles';
+import template from './template';
+
+@tagName('')
+@layout(template, styles)
+export default class RadioButton extends Component {
```

Comment:
Can this be called something else? It's using a package called `RadioButton` so it was a bit confusing looking at it. `FormControlRadioButton`? `RadioSingleSelect`?

Reply:
Maybe better named as `FormControlSingleSelect` because we care about it being a "single select" widget instead of a "radio button" widget.